contrib/starrocks-python-client/starrocks/sql/dml.py

```diff
@@ -0,0 +1,563 @@
+        self.target = target
+        self.from_ = from_
+        self.columns = columns
+
+    def __repr__(self):
+        """
+        repr for debugging / logging purposes only. For compilation logic, see
+        the corresponding visitor in dialect.py
+        """
+        return (
+            f"INSERT INTO {self.target}"
+            f" SELECT {','.join(self.columns) if isinstance(self.columns, list) else self.columns}"
+            f" FROM {repr(self.from_)}"
+        )
+
+    def bind(self):
+        return None
+
+
+class FilesTarget(FilesClause):
+    __visit_name__ = 'files_target'
+
+    def __init__(
+        self,
+        *,
+        storage: "_StorageClause",
+        format: "FilesFormat",
+        options: "FilesTargetOptions" = None,
+    ):
+        super().__init__(format, options)
+        self.storage = storage
+
+    def __repr__(self):
+        """
+        repr for debugging / logging purposes only. For compilation logic, see
+        the corresponding visitor in dialect.py
+        """
+
+        return f"FILES(\n{repr(self.storage)}\n{repr(self.format)}\n{repr(self.options)}\n)"
+
+class FilesTargetOptions(_FilesOptions):
+    def __init__(
+        self,
+        *,
+        single: bool = None,
+        target_max_files_size: int = None,
+        partitioned_by: str = None,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        if single is not None:
+            self.options["single"] = "true" if single else "false"
+        if target_max_files_size is not None:
+            self.options["target_max_files_size"] = target_max_files_size
+        if partitioned_by is not None:
+            self.options["partitioned_by"] = partitioned_by
+
+
+class FilesSource(FilesClause):
+    __visit_name__ = 'files_source'
+
+    def __init__(
+        self,
+        *,
+        storage: "_StorageClause",
+        format: "FilesFormat",
+        options: "FilesSourceOptions" = None,
+    ):
+        super().__init__(format, options)
+        self.storage = storage
+
+    def __repr__(self):
+        """
+        repr for debugging / logging purposes only. For compilation logic, see
+        the corresponding visitor in dialect.py
+        """
+
+        return f"FILES(\n{repr(self.storage)}\n{repr(self.format)}\n{repr(self.options)}\n)"
+
+class FilesSourceOptions(_FilesOptions):
+    def __init__(
+        self,
+        *,
+        list_files_only: bool = None,
+        list_recursively: bool = None,
+        columns_from_path: str = None,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        if isinstance(list_files_only, bool):
+            self.options["list_files_only"] = str(list_files_only).lower()
+        if isinstance(list_recursively, bool):
+            self.options["list_recursively"] = str(list_recursively).lower()
+        if columns_from_path is not None:
+            self.options["columns_from_path"] = columns_from_path
+
+class Compression(Enum):
```

Comment:
It doesn't need to be so detailed, we can use general forms, and let StarRocks to judge whether the SQL is syntax connrect or not.
Otherwise, this project needs to change a lot and time to time.

Reply:
As long as there is no error for this python package, StarRocks will print errors if users have specified correct parameters. Then they can refer to the StarRocks' docs.